### ISSUE TITLE
Catch duplicate migration names

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/avram#521]
Right now you can do 

```
lucky gen.migration CreateUsers
lucky gen.migration CreateUsers
```

and this will generate 2 separate migrations called `_create_users.cr` but with different timestamps. We should raise an error on the second one saying that you've already done this. We currently do the same for generating models. 

Ref: https://discord.com/channels/743896265057632256/743896265057632259/775771902979080204

**Comment by matthewmcgarvey** [comment]
It could also be copy/paste errors, I wonder if there's a way we could detect duplicate classes. I suspect we'd only be able to check file names?

**Comment by jwoertink** [comment]
Yeah, the filenames minus the timestamps would probably be fine. I know I've done it in the past with rails where I've done "rails g migration add_files_to_users" then like 3 years later ran the same thing...  It's laziness, but maybe we can help catch that to force you to be a little more explicit.

**Comment by wontruefree** [comment]
I think raising a warning might be more appropriate unless there could be unexpected behavior.

**Comment by jwoertink** [comment]
This is fixed in #528
